builder/comp-builder.nix

```diff
@@ -146,7 +146,12 @@ let
 
   disableFeature = disable: enableFeature (!disable);
 
-  finalConfigureFlags = lib.concatStringsSep " " (
+  patchInstLine = instLine:
+    if lib.strings.hasInfix "inplace" instLine then
+      "$(read packageId < <(read sedline < <(echo ${instLine} | sed -n -e 's/.*=\\(.*\\)-inplace-\\(.*\\):.*/\\1-\\\\\\\\([^\"]*\\\\\\\\)-\\2/p') ; sed -n -e \"s/.*\$\{sedline\}/\\1/p\" ${configFiles}/configure-flags | head -1) ; echo ${instLine} | sed -n -e \"s/\\(.*\\)-inplace-\\(.*\\)/\\1-\$\{packageId\}-\\2/p\")"
+    else instLine;
```

Comment:
This needs a comment, explaining what we try to rewrite, from what to what and how.